flatliners/versionedmetrics.py

```diff
@@ -1,12 +1,11 @@
 from .baseflatliner import BaseFlatliner
 
-
 class VersionedMetrics(BaseFlatliner):
     def __init__(self):
         super().__init__()
 
         # dict for versions that we already know
-        self.cluster_versions = dict()
+        self.cluster_versions = self.create_cache_dict()
```

Comment:
please add a comment about the expected size of this dict and how to find out what size is best.
and what is stored in the dict. maybe an example

Reply:
@4n4nd 

> please add a comment about the expected size of this dict and how to find out what size is best.

Reply:
```python
 # caches the version for a specific cluster_id, e.g. cluster_versions['_id_123'] = 'version4'
 # the expected size the number of clusters reporting into telemeter
 # promql: count (count by (_id) (cluster_version))
 self.cluster_versions = self.create_cache_dict(max_size = 1000)
```